### PR TITLE
wrapped get item in retry as temporary measure

### PR DESCRIPTION
**Pull request by EvanTedesco**

### CODE REVIEW

File: app/models/item_lookup.rb [hunk 1]
```diff
@@ -8,12 +8,18 @@ def initialize(sku)
 
   def get_item(sku)
     tries = 0
-    while(current_price.nil? && tries < 20)
-      @item = client.lookup(sku).first
-      tries += 1
+    while current_price.nil?
```
Comment (stevepm) [inline app/models/item_lookup.rb]: wouldn't you need && tries < 20?

Comment (EvanTedesco) [inline app/models/item_lookup.rb]: retry unless tries >= 20

Comment (EvanTedesco) [inline app/models/item_lookup.rb]: because we only need to retry if it fails.  If it succeeds then current_price != nil

Comment (stevepm) [inline app/models/item_lookup.rb]: it won't retry, instead the code would break

**Pull request merged**